functorch/_src/aot_autograd.py

```diff
@@ -397,14 +397,6 @@ def fake_fn(primals, tangents):
         with track_graph_compiling("forward"):
             compiled_fw_func = aot_config.fw_compiler(fw_module, deduped_flat_args)
 
-        if config.debug_partitioner:
-            fw_outs = call_func_with_args(compiled_fw_func, deduped_flat_args)
-            activation_sizes = 0
-            for out in fw_outs[_num_outs:]:
-                if isinstance(out, torch.Tensor):
-                    activation_sizes += out.storage().nbytes()
```

Comment:
`activation_sizes += out.storage().size() * out.element_size()` ? 

alternatively add `nbytes` to TypedStorage